### ISSUE TITLE
[CogBoard] Switch to using Red's embed requested API

### ISSUE THREAD

**Issue by Jackenmen** [jack1142/JackCogs#26]
I didn't use embed requested API, because of lack of granular settings. While per-channel settings mostly addressed that, per-command embed settings would be perfect.

Blocked by Cog-Creators/Red-DiscordBot#4049

**Comment by Jackenmen** [comment]
This issue is no longer relevant due to pending #38
